src/main/java/by/a1qa/data/TestData.java

```diff
@@ -0,0 +1,11 @@
+package by.a1qa.data;
+
+public class TestData {
+    public static final String firstTitle = "Get tickets at a price you’ll brag about.";
+    public static final String secondTitle = "Book in a few taps for today, tomorrow, or months from now.";
+    public static final String thirdTitle = "Deals that seem too good to be true.";
+    public static final String fourthTitle = "Find great shows near you.";
+    public static final String fifthTitle = "Join us!";
+    public static final String cityName = "London";
```

Comment:
`cityName` should not belong here.
while I can accept other variables here, because text is pretty static (the class name is awful though), the vity used it tests is no way static. It should be stored either in the tests themselves, or in property file under `defaultCity` property.